docs/section5/Rule5-39.md

```diff
@@ -0,0 +1,45 @@
+
+# Envelope - Rule 5-39  
+
+**Rule ID:** 5-39  
+**Rule Description:** Automatically controlled dynamic glazing may be modeled. Manually controlled dynamic glazing shall use the average of the minimum and maximum SHGC and VT.  
+**Rule Assertion:** P-RMR subsurface: is_dynamic_glazing = expected value  
```

Comment:
Ths assertion should be that subsurface:solar_heat_gain_coefficient, subsurface:visible_transmittance=expected value

Reply:
This comment is not picked up as per the latest schema change.